src/datapump/jobs/version_update.py

```diff
@@ -229,6 +253,30 @@ def _create_cog_asset(self, cog_asset_parameters: CogAssetParameters) -> str:
         data = client.create_aux_asset(self.dataset, self.version, payload)
 
         return data["asset_id"]
+    
+    def _create_aggregated_tile_set(self, aggregated_tile_set_parameters: AuxTileSetParameters):
+        """
+        Create aggregated tile set
+        """
+        client = DataApiClient()
+
+        co = aggregated_tile_set_parameters
+
+        assets = client.get_assets(self.dataset, self.version)
+
+        payload = {
+            "asset_type": "Aggregated tile set",
+            "creation_options": {
+                "calc": co.calc,
+                "grid": co.grid,
+                "data_type": co.data_type,
+                "no_data": co.no_data,
+                "pixel_meaning": co.pixel_meaning,
+                "auxiliary_assets": co.auxiliary_assets
```

Comment:
**Maybe you haven't gotten to this yet, but you need to do the same lookup here based on co.auxiliary_assets_pixel_meaning that you are doing for _create_aux_tile.set.

Reply:
Does it makes sense to just use `_create_aux_tile_set` for both the aggregated tile set and intensity? The only difference is that the aggregated tile set needs last week's version for its calc and the intensity tile set needs this week's version for its calc, but that is handled above because we the step order is `creating_aggregated_tile_set` then `mark_latest` then `creating_aux_assets` (latter for intensity). Am I missing something here?

Reply:
Yes, from what I can see, you can combine them - that's nice.  We just need to make some very obvious comments that creating_aggregated_tile_set must occur before mark_latest and creating_aux_assets must happen after mark_latest.